presto-jdbc/src/main/java/io/prestosql/jdbc/PrestoParameterMetaData.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.jdbc;
+
+import com.google.common.collect.ImmutableList;
+
+import java.sql.ParameterMetaData;
+import java.sql.SQLException;
+import java.sql.Types;
+import java.util.List;
+
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+
+public class PrestoParameterMetaData
+        implements ParameterMetaData
+{
+    private final List<ColumnInfo> parameterInfo;
+
+    PrestoParameterMetaData(List<ColumnInfo> parameterInfo)
+    {
+        this.parameterInfo = ImmutableList.copyOf(requireNonNull(parameterInfo, "parameterInfo is null"));
+    }
+
+    @Override
+    public int getParameterCount() throws SQLException
+    {
+        return parameterInfo.size();
+    }
+
+    @Override
+    public int isNullable(int param) throws SQLException
+    {
+        switch (getParameter(param).getNullable()) {
+            case NO_NULLS:
+                return ParameterMetaData.parameterNoNulls;
```

Comment:
```suggestion
                return parameterNoNulls;
```
similarly for others too